agent/consul/discoverychain/gateway.go

```diff
@@ -198,9 +198,12 @@ func (l *GatewayChainSynthesizer) consolidateHTTPRoutes() []structs.HTTPRouteCon
 	return consolidateHTTPRoutes(l.matchesByHostname, l.suffix, l.gateway)
 }
 
-// ReformatHTTPRoute takes in an HTTPRoute and reformats it to match the discovery chains generated by the gateway chain synthesizer
-func ReformatHTTPRoute(route *structs.HTTPRouteConfigEntry, listener *structs.APIGatewayListener, gateway *structs.APIGatewayConfigEntry) []structs.HTTPRouteConfigEntry {
-	matches := initHostMatches(listener.GetHostname(), route, map[string][]hostnameMatch{})
+// ConsolidateHTTPRoutes takes in one or more HTTPRoute and reformats them to match the discovery chains generated by the gateway chain synthesizer
```

Comment:
I wound up rewriting this docstring in my 1.16 PR to this after noticing @missylbytes 's comment earlier
```suggestion
// ConsolidateHTTPRoutes takes in one or more HTTPRoutes and consolidates them down to the minimum
// set of HTTPRoutes that can represent the same set of rules. This should result in approx. one
// HTTPRoute per hostname.
```

Reply:
updated with new text